scrapy/shell.py

```diff
@@ -146,14 +146,13 @@ def get_help(self):
         b.append("Useful shortcuts:")
         if self.inthread:
             b.append("  fetch(url[, redirect=True]) "
-                     "Fetch URL and update local objects "
-                     "(by default, redirects are followed)")
+                     "Fetch URL and update local objects (by default, redirects are followed)")
             b.append("  fetch(req)                  "
                      "Fetch a scrapy.Request and update local objects ")
         b.append("  shelp()           Shell help (print this help)")
         b.append("  view(response)    View response in a browser")
 
-        return "\n".join("[s] %s" % l for l in b)
+        return "\n".join("[s] %s" % line for line in b)
```

Comment:
Hi @elacuesta 

Maybe it's not necessary to do it as part of this PR, but as we are giving more meaning to a lot of variable names, it could be a good idea to rename this `b` variable to something like `banner` (I assume this is the real meaning by looking at this: `self.vars['banner'] = self.get_help()`).

Reply:
Indeed, I almost did that, but I decided not to in order to keep the diff simpler. Perhaps we can do it in a later PR :+1: